indexer/search_string_utils.cpp

```diff
@@ -7,6 +7,8 @@
 
 #include "3party/utfcpp/source/utf8/unchecked.h"
 
+#include <algorithm>
```

Comment:
Are you sure that we need `#include <algorithm>`?

Reply:
Yes, we use swap() here.